src/shogun/classifier/svm/LibSVMOneClass.h

```diff
@@ -21,6 +21,10 @@ namespace shogun
 class CLibSVMOneClass : public CSVM
 {
 	public:
+
+		/** problem type */
+		MACHINE_PROBLEM_TYPE(PT_CLASS);
```

Comment:
not sure about this. does PT_CLASS means its one class classification?

Reply:
yes

Reply:
it is a binary classifier

Reply:
any idea why we have `PT_BINARY` and `PT_CLASS`?

Reply:
`PT_CLASS` doesn't have its own `apply` version, and it makes tests fail if used.